diff --git a/data/systemd/snapd.service.in b/data/systemd/snapd.service.in
index cb00c9e..04ff7c8 100644
--- a/data/systemd/snapd.service.in
+++ b/data/systemd/snapd.service.in
@@ -7,7 +7,7 @@ Requires=snapd.socket
 # (https://bugs.launchpad.net/snapd/+bug/1709536)
 #Nice=-5
 OOMScoreAdjust=-900
-ExecStart=@libexecdir@/snapd/snapd
+ExecStart=/usr/lib64/snapd
 EnvironmentFile=-@SNAPD_ENVIRONMENT_FILE@
 Restart=always
 Type=notify
