diff -Naur a/steam b/steam
--- a/steam	2016-08-14 18:34:25.911287460 +0100
+++ b/steam	2016-08-14 18:38:10.673046304 +0100
@@ -31,6 +31,13 @@
 	[[ -n "${LDPATH}" ]] && LD_LIBRARY_PATH+=":${LDPATH}"
 fi
 
+# Set a default STEAM_RUNTIME value.
+: ${STEAM_RUNTIME:=@@STEAM_RUNTIME@@}
+export STEAM_RUNTIME
+# Set a default STEAM_RUNTIME_PREFER_HOST_LIBRARIES value.
+: ${STEAM_RUNTIME_PREFER_HOST_LIBRARIES:=@@STEAM_RUNTIME_PREFER_HOST_LIBRARIES@@}
+export STEAM_RUNTIME_PREFER_HOST_LIBRARIES
+
 function show_message()
 {
style=$1
