--- a/setup.py	2019-08-12 19:27:53.396861000 +0100
+++ b/setup.py	2019-08-12 19:41:28.833259045 +0100
@@ -708,9 +708,9 @@
         run_tool(['make', 'docs'])
     copy_man_pages(ddir)
     copy_html_docs(ddir)
-    icdir = os.path.join(ddir, 'share', 'icons', 'hicolor', '256x256', 'apps')
+    icdir = os.path.join(ddir, 'share', 'icons', 'hicolor', 'scalable', 'apps')
     safe_makedirs(icdir)
-    shutil.copy2('logo/kitty.png', icdir)
+    shutil.copy2('logo/kitty.svg', icdir)
     deskdir = os.path.join(ddir, 'share', 'applications')
     safe_makedirs(deskdir)
     with open(os.path.join(deskdir, 'kitty.desktop'), 'w') as f:
