--- a/ctdb/config/functions	2015-05-27 22:47:46.000000000 +0300
+++ b/ctdb/config/functions	2015-08-08 19:21:58.000000000 +0300
@@ -894,6 +894,7 @@ ip_maskbits_iface ()
     esac
 
     ip addr show to "${_addr}/${_bits}" 2>/dev/null | \
+	sed -e 's/@NONE:/:/' | \
 	awk -v family="${_family}" \
 	    'NR == 1 { iface = $2; sub(":$", "", iface) } \
              $1 ~ /inet/ { mask = $2; sub(".*/", "", mask); \
--- a/ctdb/tests/scripts/integration.bash	2015-05-27 22:47:46.000000000 +0300
+++ b/ctdb/tests/scripts/integration.bash	2015-08-08 19:21:46.000000000 +0300
@@ -698,6 +698,7 @@ ip_maskbits_iface ()
     esac
 
     ip addr show to "${_addr}/${_bits}" 2>/dev/null | \
+	sed -e 's/@NONE:/:/' | \
 	awk -v family="${_family}" \
 	    'NR == 1 { iface = $2; sub(":$", "", iface) } \
              $1 ~ /inet/ { mask = $2; sub(".*/", "", mask); \
