Description: Use Noto font instead of Droid.
Author: Bas Couwenberg <sebastic@debian.org>
Bug-Debian: https://bugs.debian.org/804683
Forwarded: https://josm.openstreetmap.de/ticket/12085

--- a/src/org/openstreetmap/josm/gui/mappaint/styleelement/StyleElement.java
+++ b/src/org/openstreetmap/josm/gui/mappaint/styleelement/StyleElement.java
@@ -162,7 +162,7 @@ public abstract class StyleElement imple
             synchronized (lock) {
                 n = defaultFontName;
                 if (n == null) {
-                    defaultFontName = n = Config.getPref().get("mappaint.font", "Droid Sans");
+                    defaultFontName = n = Config.getPref().get("mappaint.font", "Noto Sans");
                 }
             }
         }
